### PR TITLE
social map about page (updated with Shai's details)

### PR DESCRIPTION
**Pull request by noamoss**

### CODE REVIEW

File: abouts/socialmap.md [hunk 1]
```diff
@@ -60,10 +60,10 @@
 
 ## שותפים במיזם {supporters}
 
-#### [![הסדנא לידע ציבורי](assets/img/hasadna.png =120x120)](https://hasadna.org.il)  [![מידות](assets/img/midot.svg =120x72)](http://www.midot.org.il/)   [![חברת מגדל](assets/img/migdal.svg =120x72)](https://www.migdal.co.il/)
```
Comment (akariv) [inline abouts/socialmap.md]: why?

Comment (noamoss) [reply abouts/socialmap.md]: what do you mean by "why"? these are our partners... isn't it so?

**Comment by noamoss** [comment]
@akariv for an unclear reason the master branch was not updated with the recent content. Should I delete and push again only Shai's title?

**Comment by akariv** [comment]
You can simply add another commit that reverts that line to its correct state.

File: abouts/socialmap.md (hunk 1 above)
Comment (noamoss) [reply abouts/socialmap.md]: @akariv did not you tell me we should add #### to centralise the icons on this page?

**Pull request merged**